src/status_im/ui/screens/routing/core.cljs

```diff
@@ -38,7 +38,10 @@
       (when-not modal?
         (status-bar/set-status-bar current-view-id)))
     :on-will-blur
-    (fn [] (reset! screen-focused? false))}])
+    (fn []
+      (reset! screen-focused? false)
+      (doseq [text-input @react/text-input-refs]
+        (.clear text-input)))}])
```

Comment:
if ref is nil, is it safe to call crear ? a mean maybe we have to remove nil refs from list?

Reply:
text-input makes sure that only non-nil refs are added to `text-input-refs`.